service/src/main/java/bio/terra/pipelines/common/utils/PipelinesEnum.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.pipelines.common.utils;
+
+public enum PipelinesEnum {
+  IMPUTATION("imputation");
```

Comment:
we should probably make a ticket to clean out the toy pipelines in the pipelines table.

Reply:
cleaned up in this PR, thanks for the reminder